play/src/components/joingame/EnterPlayerName.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { styled, useTheme } from '@mui/material/styles';
+import { Stack, Box, Grid, Typography } from '@mui/material';
+import {
+  IntroButton,
+  IntroTextField,
+  JoinGameBackgroundContainer,
+} from '../../lib/styledcomponents/StyledComponents';
+import Logo from '../../img/rightOnLogo.svg';
+
+const StackContainer = styled(Stack)(({ theme }) => ({
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  maxWidth: theme.breakpoints.values.xs,
+}));
+
+const PaddedContainer = styled(Box)(({ theme }) => ({
+  paddingLeft: `${theme.sizing.smallPadding}px`,
+  paddingRight: `${theme.sizing.smallPadding}px`,
+}));
+
+interface EnterPlayerNameProps {
+  playerFirstName: string;
+  playerLastName: string;
+  handlePlayerFirstNameChange: (newValue: string) => void;
+  handlePlayerLastNameChange: (newValue: string) => void;
+  inputError: boolean;
+}
+
+export default function EnterPlayerName({
+  playerFirstName,
+  playerLastName,
+  handlePlayerFirstNameChange,
+  handlePlayerLastNameChange,
+  inputError,
+}: EnterPlayerNameProps) {
+  const theme = useTheme();
+  return (
+    <JoinGameBackgroundContainer>
+      <StackContainer spacing={5}>
+        <img
+          style={{
+            width: '214px',
+            height: '118px',
+            paddingTop: `${theme.sizing.extraLargePadding}px`,
+          }}
+          src={Logo}
+          alt="Question"
+        />
+        <PaddedContainer>
+          <Typography variant="h2" sx={{ textAlign: 'center' }}>
+            Enter Your Name
+          </Typography>
+          <Grid container spacing={2} wrap="nowrap">
+            <Grid item xs={6}>
+              <IntroTextField
+                fullWidth
+                variant="filled"
+                autoComplete="off"
+                onChange={(newValue) => {
+                  handlePlayerFirstNameChange(newValue.target.value);
+                }}
+                onFocus={(newValue) => {
+                  if (newValue.target.value === 'First Name') {
```

Comment:
Same comment similar to the other PR, probably wanna extract `'First Name'` into a const and then use that instead of hardcoding string across the file

Reply:
Also, same comment regarding `placeholder` usage, looks like we use `onFocus` to remove the default value in the field while if we use `placeholder` we shouldn't need this.

And if we do this, then you can use something like `input::placeholder` or there should be a similar approach to modify the text color for placeholder vs. normal message

Reply:
placeholder values have been moved to an enum in `/lib/PlayModels` and placeholder values are fixed per changes in HowToPlay.